Testing/Data/UnitTest/HRP39180.RAW.md5

```diff
@@ -0,0 +1 @@
+7d19937fbfb68e962c190454a128fde6
```

Comment:
Can we avoid adding a new copy of this file hash? We'll now have the same file in `UnitTest`, `DocTest` and `SystemTest` if we add it here and if it ever needs to be updated we have to remember it is in 3 places.

Unit tests/ system tests have access to DocTest so can we just leave the copy that is in there and remove this and the one in `Data/SystemTest?`

I think this has also caused the seemingly strange failure of the doctests for LoadRaw3. The `UnitTest` directory is first in the search path so this file would now be picked up. The RAW file format does not have all log data contained within the file but relies on separate `*.txt` files alongside the `.raw` to load additional log data. In `Data/DocTest` there are 3 files related to HRP39180:
```
ExternalData/Testing/Data/DocTest/HRP39180_INT1.txt
ExternalData/Testing/Data/DocTest/HRP39180_INT2.txt
ExternalData/Testing/Data/DocTest/HRP39180.RAW
```
The .raw file gives 33 logs and then the 2 extra .txt files give bring the total to 35. By copying this file to UnitTest but without the logs it now doesn't see the extra `.txt` files so gives you back only 33. When you switch back to master you will still have the file in the downloaded 'ExternalData/Testing/Data/UnitTest' directory so master will seem like it fails too. For this reason we always remove the links to the downloaded ExternalData files on the [buildservers](https://github.com/mantidproject/mantid/blob/master/buildconfig/Jenkins/buildscript#L200). If you do the same then master should start working again.

The upshot here is I think if we remove this file from `UnitTest` and ``SystemTest` then I think that will be fine.

Reply:
:+1:

Reply:
@martyngigg it worked. Thanks a million